molecule/default/verify.yml

```diff
@@ -4,6 +4,21 @@
 - name: Verify
```

Comment:
@evansmurithi do we still need this file?

Reply:
No, forgot to remove it